apps/docs/src/app/core/component-docs/moment-datetime-adapter/moment-datetime-adapter-docs.component.html

```diff
@@ -0,0 +1,61 @@
+<fd-docs-section-title
+    id="simple"
+    componentName="momentDatetimeAdapter"
+>
+    Date Picker example with MomentDatetimeAdapter
+</fd-docs-section-title>
+<description>
+    DateAdapter is a generic class, components such as Datepicker also need to be made generic.
+    When working with these classes (for example as a ViewChild) you should include the appropriate generic type
+    that corresponds to the DateAdapter implementation you are using. For example:
+    <br>
+    <br>
+    <code>@ViewChild(DatePickerComponent) datePicker: DatePickerComponent&lt;<strong>Moment</strong>&gt;</code>;
+</description>
+<component-example>
+    <fd-date-picker-moment-adapter-example></fd-date-picker-moment-adapter-example>
+</component-example>
+<code-example [exampleFiles]="datePickerMomentAdapter"></code-example>
+
+<separator></separator>
+
+<fd-docs-section-title
+    id="moment-datetime-options"
+    componentName="momentDatetimeAdapter"
```

Comment:
```suggestion
    componentName="moment-datetime-adapter"
```
anchor links don't work currently. Similar change at [line 3](https://github.com/SAP/fundamental-ngx/pull/6000/files#diff-aacef743fba6cc912365b3e92d2846af0f880e3b545ffe9297450045b47da147R3) and [line 42](https://github.com/SAP/fundamental-ngx/pull/6000/files#diff-aacef743fba6cc912365b3e92d2846af0f880e3b545ffe9297450045b47da147R42)